jdaviz/configs/mosviz/plugins/viewers.py

```diff
@@ -88,8 +88,9 @@ def __init__(self, *args, **kwargs):
             "spectrum_2d_viewer_reference_name", "spectrum-2d-viewer"
         )
 
-    def data(self, cls=None):
-        return [layer_state.layer.get_object(cls=cls or self.default_class)
+    def data(self, cls=None, statistic=None):
+        return [layer_state.layer.get_object(statistic=statistic,
```

Comment:
Is this an unrelated bug fix? If so, the change log also need to reflect this fix.

Actually I don't see any change log modification here, so that has to be fixed too.